src/components/PlioQuestion.vue

```diff
@@ -218,13 +227,88 @@ export default {
           this.hideSkipButton();
           this.isAnswerSubmitted = true;
           this.disableRadioButtons();
-        }, 200);
+        });
       }
 
       this.text = "";
       this.show = true;
       document.querySelector("body").classList.add("overflow-hidden");
       this.newProgressBarInfo['progressPercent'] = this.progressBarInfo['progressPercent']
+      this.handleImage()
+      this.styleQuestion()
+      this.styleOptions()
+      this.renderContent()
+    },
+
+    handleImage(){
+      this.$nextTick(() => {
+        // get question element
+        var question = document.getElementById('question')
+        if (question == null) return
+        // extract all <p> tags from the question element
+        var p_tags = question.querySelectorAll('p')
+        if (p_tags == null || p_tags.length == 0) return
+        // find which <p> tag has an <img> tag, cut it and
+        // paste it right outside, just below it.
+        p_tags.forEach(p_tag => {
+          var img_tag = p_tag.querySelector('img')
+          if (img_tag != null) {
+            p_tag.removeChild(img_tag)
+            p_tag.parentNode.insertBefore(img_tag, p_tag.nextSibling)
+            img_tag.setAttribute("id", "question_image")
+            // scale the image such that width is 1/3rd of screen width
+            // maintaining the aspect ratio
+            var heightToWidth = (img_tag.offsetHeight/img_tag.offsetWidth)
```

Comment:
better?
```suggestion
            var aspectRatio = (img_tag.offsetHeight/img_tag.offsetWidth)
```

Reply:
Aspect Ratio is width/height (not height/width). but otherwise yes, use aspect ratio!

Reply:
Renamed the variable to aspectRatio

Reply:
It's not aspect ratio. Aspect ratio is `width/height`, not `height/width` ... so if you're going with aspect ratio, change teh calculation and use the new value

Reply:
Oh I thought you said that I should just change the variable name :P

Reply:
Done. Committing